contracts/deprecated/CrossChain.sol

```diff
@@ -23,17 +17,17 @@ contract CrossChain is System, ICrossChain, IParamSubscriber {
     uint256 public constant INIT_BATCH_SIZE = 50;
 
     // governable parameters
-    uint256 public batchSizeForOracle;
+    uint256 public batchSizeForOracle;  // @dev deprecated
```

Comment:
I think `encodePayload` can be deleted, no one is using it.

Reply:
removed